src/GeneratorV2/Process/Overloaders/PointerToOffsetOverloader.cs

```diff
@@ -0,0 +1,105 @@
+using System.Collections.Generic;
+using System.Linq;
+using GeneratorV2.Writing;
+
+namespace GeneratorV2.Process.Overloaders
+{
+    public sealed class PointerToOffsetOverloader : IOverloader
+    {
+        private readonly Dictionary<string, string> _methodsAndParametersToOverload = new Dictionary<string, string>
+        {
+            // Draw elements permutations.
+            {"glDrawElements", "indices"},
+            {"glDrawElementsBaseVertex", "indices"},
+            {"glDrawElementsBaseVertexEXT", "indices"},
+            {"glDrawElementsBaseVertexOES", "indices"},
+            {"glDrawElementsInstanced", "indices"},
+            {"glDrawElementsInstancedANGLE", "indices"},
+            {"glDrawElementsInstancedARB", "indices"},
+            {"glDrawElementsInstancedBaseInstance", "indices"},
+            {"glDrawElementsInstancedBaseInstanceEXT", "indices"},
+            {"glDrawElementsInstancedBaseVertex", "indices"},
+            {"glDrawElementsInstancedBaseVertexBaseInstance", "indices"},
+            {"glDrawElementsInstancedBaseVertexBaseInstanceEXT", "indices"},
+            {"glDrawElementsInstancedBaseVertexEXT", "indices"},
+            {"glDrawElementsInstancedBaseVertexOES", "indices"},
+            {"glDrawElementsInstancedEXT", "indices"},
+            {"glDrawElementsInstancedNV", "indices"},
+            {"glDrawRangeElements", "indices"},
+            {"glDrawRangeElementsBaseVertex", "indices"},
+            {"glDrawRangeElementsBaseVertexEXT", "indices"},
+            {"glDrawRangeElementsBaseVertexOES", "indices"},
+            {"glDrawRangeElementsEXT", "indices"},
+            // {"glMultiDrawElements", "indices"},
+            // {"glMultiDrawElementsBaseVertex", "indices"},
+            // {"glMultiDrawElementsBaseVertexEXT", "indices"},
+            // {"glMultiDrawElementsEXT", "indices"},
+            // {"glMultiModeDrawElementsIBM", "indices"},
```

Comment:
These are commented out because they use an array of offsets, which currently isnt being handled entirely correct. This should be pretty easy to fix. Just didnt have the time yesterday.

Reply:
Added a FIXME in the code that says this too.